ui/src/pages/Settings.vue

```diff
@@ -3,7 +3,7 @@
     <template #header>
       <PageHeading :crumbs="crumbs">
         <template #actions>
-          <p-key-value label="Version" :value="version" alternate />
+          <p-key-value class="settings__version" label="Version" :value="version" alternate />
```

Comment:
I'm surprised this is the solution - I'd expect `PageHeading` to be responsible for floating the `actions` slot to the right

Reply:
So page has that as `fit-content`. I also tried `min-content` but that didn't help. p-key-value has `width: 100%` which is the culprit here. I'm not sure why it does tbh. But we're not having any issues what that component elsewhere so adding `width: auto` here seemed like the simplest fix.

Reply:
Fair enough, this is certainly the least-risky change 👍🏻